practice-app/website/api/event.py

```diff
@@ -24,29 +28,29 @@ def getCoordinates(address):
             longitude = result['results'][0]['geometry']['location']['lng']
             latitude = result['results'][0]['geometry']['location']['lat']
             return formatted_address, longitude, latitude, "OK"
-        
-    if result['status'] == "ZERO_RESULTS" or result['status'] == "INVALID_REQUEST": 
-        return "",0,0,"Address Not Valid"
-    elif result['status'] == "OVER_DAILY_LIMIT" or result['status'] == "OVER_QUERY_LIMIT" or result['status'] == "REQUEST_DENIED" or result['status'] == "UNKNOWN_ERROR": 
-        return "",0,0,"Try Later"
-    else:
-        return "",0,0,"Try Later"
 
-    
+    if result['status'] == "ZERO_RESULTS" or result['status'] == "INVALID_REQUEST":
+        return "", 0, 0, "Address Not Valid"
+    elif result['status'] == "OVER_DAILY_LIMIT" or result['status'] == "OVER_QUERY_LIMIT" or result['status'] == "REQUEST_DENIED" or result['status'] == "UNKNOWN_ERROR":
+        return "", 0, 0, "Try Later"
+    else:
+        return "", 0, 0, "Try Later"
 
 
-@events.route('/', methods = ['GET','POST'])
+@events.route('/', methods=['GET', 'POST'])
 @swag_from('doc/events_POST.yml', methods=['POST'])
 def event():
+
     if request.method == 'GET':
         eventList = Event.query.all()
         return jsonify([event_item.serialize() for event_item in eventList]), 201
 
     if request.method == 'POST':
-        if not request.json or not 'name' in request.json or not 'creator_user'  in request.json or not 'location' in request.json or not 'sport' in request.json or not 'date' in request.json:
+        if not request.json or not 'name' in request.json or not 'creator_user' in request.json or not 'location' in request.json or not 'sport' in request.json:
```

Comment:
This should not be changed

Reply:
I think this point can be resolved during merge operation, too.